aptos-move/move-examples/fungible_asset/sources/managed_fungible_asset.move

```diff
@@ -0,0 +1,216 @@
+module fungible_asset::managed_fungible_asset {
```

Comment:
Shouldn't this be managed_fungible_source since it's extending the token (fungible source?) object?